data/campaigns/Heir_To_The_Throne/scenarios/20b_Underground_Channels.cfg

```diff
@@ -190,18 +183,88 @@
     [event]
         name=moveto
         [filter]
-            x=43-50
-            y=1-5
+            x=41,42,43-44,45-46,47-48,49-50
+            y=3,2,1-3,1-4,1-5,1-6
             id=Konrad
         [/filter]
 
-        [message]
-            speaker=Bona-Melodia
-            message= _ "Get that vile creature!"
-        [/message]
+        [if]
+            [have_unit]
+                id=Bona-Melodia
+            [/have_unit]
+            [then]
+                [message]
+                    speaker=Bona-Melodia
+                    message= _ "Get that vile creature!"
+                [/message]
+
+                [message]
+                    speaker=Kalenz
+                    message= _ "Hold!"
+                [/message]
+
+                [message]
+                    speaker=Bona-Melodia
+                    message= _ "Milord Kalenz! My appologies. I mistook him for one of those foul lizardmen. How came you to be traveling such paths?"
```

Comment:
Typo "appologies". Also "litte time" a bit further down. The "I see battle-sign" thing is also rather weird. It sounds more like something an ogre or saurian might say.